about.pug

```diff
@@ -8,6 +8,17 @@ block prepend title
 
 block content
     h1 About
+    p.
+        Puyo Puyo VS is a client that can connect to internet lobbies in order to play Puyo Puyo Matches against up to 10 people at the same time.
+        Features the ability to customize assets used ingame, modes such as Tsu, which is the standard Puyo ruleset, as well as Fever, Non-stop Fever, and more, and
```

Comment:
This line feels pretty good, but I would prefer if it started with either `it` or `Puyo Puyo VS`. Mainly because I feel like it would read nicer as:

```
Puyo Puyo VS features the ability...
```